packages/message-encryption/src/symmetric.ts

```diff
@@ -90,21 +93,30 @@ export interface EncoderOptions extends BaseEncoderOptions {
  * in [26/WAKU2-PAYLOAD](https://rfc.vac.dev/spec/26/).
  */
 export function createEncoder({
+  pubSubTopic = DefaultPubSubTopic,
```

Comment:
With autosharding, the user won't have to set the pubsub topic and hence it should almost disappear from the API, so I'd aim to always have it listed last.